fastlane/Fastfile

```diff
@@ -237,7 +238,8 @@ platform :ios do
     xcodebuild(
       workspace: 'RevenueCat.xcworkspace',
       scheme: 'SwiftAPITester',
-      destination: 'generic/platform=iOS Simulator'
+      destination: 'generic/platform=iOS Simulator',
+      configuration: 'release',
```

Comment:
I wonder if we should build the api tests in both debug and release, in case we have any debug-only apis... I agree release is more important though, so this makes sense.

Reply:
We compile API testers in debug when running tests.